spec.md

```diff
@@ -0,0 +1,315 @@
+
+For computer processing, artifacts are defined by setting the `manifest.config.mediaType` to a globally unique value.
+
+> **Note:** The `config.mediaType` of `application/vnd.oci.image.config.v1+json` is reserved for artifacts intended to be run and instanced by [docker][docker], [containerd][containerd] and other [OCI Image][image-spec] runtimes and tool chains.
+
+Each new artifact type MUST be uniquely defined. See [registering unique types with IANA](#registering-unique-types) for registering unique mediaTypes.
+
+The following `mediaType` format is used to differentiate the type of artifact:
+
+`[registration-tree]`.`[org|company|entity]`.`[objectType]`.`[optionalSubType]`.config.`[version]`+`[optionalConfigFormat]`
+
+- **`registration-tree`** - represents an IANA root tree. See [iana registration trees][iana-trees] for public and private tree options. 
+- **`org|company|entity`** - represents an open source foundation (`oci`, `cncf`) or a company (`microsoft`, `ibm`) or some unique entity.
+- **`objectType`** - a value representing the short name of the type.
+- **`optionalSubType`** - provides additional extensibility of an `objectType`
+- **`version`** - provides artifact authors the ability to revision their schemas and formatting, enabling tools to identify which format they will process.
+- **`optionalConfigFormat`** - while `config.mediaType` represents the unique identifier of an artifact, providing a config object is optional. If a config object is provided. `.json`, `.yaml|yml` or other textual formats are preferred enabling optional, but optimized processing by registry operators upon ingress.
+
+See [visualizing artifacts](#visualizing-artifacts) for `mediaType` examples.
+
+## Defining Supported Layer Types
+
+The content of an artifact is represented through a collection of layers and an optional config object. The layers of an Artifact may be independent files, collections of files, or ordinal layered tarballs persisted as blobs within a registries storage provider.
+
+Defining artifact layers involves:
+
+1. [Defining layer content, understanding layer de-duping](#layer-content-format)
+1. [Optional versioning of layer content](#layer-versioning)
+1. [Defining unique `layer.mediaTypes`](#defining-layermediatypes)
+
+As an example, [OCI Images][image-layer] are represented through an ordinal collection of tar archives. Each blob represents a layer. Each layer overlays the previous layer.
+
+Some artifacts may be represented as a single file, such as config artifact representing a deployment specification. While other artifacts may include a config object, and a collection of binaries compressed as yet another layer. By separating the layers, the artifact author can benefit from layer de-duplication and concurrent downloading of each layer using various distribution-spec clients.
+
+### Layer Content Format
```

Comment:
We should add a note here that it is up to the clients to determine how to apply/merge the layers based on the specified format